README.md

```diff
@@ -1,4 +1 @@
-# VeriYapilariOdev
-Veri yapıları dersinin ödevinde yaptığınız kodları veya örnek bir kodu pull request yapınız.
 
```

Comment:
#include<conio.h>
#include<math.h>
#include <stdio.h>
 
#define MAXCHAR 1000
int main() {
    FILE *fp;
    char str[MAXCHAR];
    char* filename = "c:\\Users\\hakan\\Downloads\\veri.txt";
 
    fp = fopen(filename, "r");
    if (fp == NULL){
        printf("Could not open file %s",filename);
        return 1;
    }
    while (fgets(str, MAXCHAR, fp) != NULL)
        printf("%s", str);
    fclose(fp);
    return 0;
}